Functions-request-config.js

```diff
@@ -5,6 +5,7 @@ require("dotenv").config()
 
 const Location = {
   Inline: 0,
+  Remote: 1,
```

Comment:
The plugin project uses https://github.com/dethcrypto/TypeChain to auto-generate all typings from Solidity code. This is amazing property and benefit of using TypeScript (someday). FYI.